src/main/java/Todo.java

```diff
@@ -0,0 +1,15 @@
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super(description);
+       // System.out.println(String.format(" [%s] [%s] %s ", 'T', " ", description));
+
+
```

Comment:
Perhaps you can remove this commented code which is not used; you can always check the revision history if you need it again